pbs_light.go

```diff
@@ -142,6 +143,64 @@ func getAccountMetrics(id string) *AccountMetrics {
 	return am
 }
 
+type cookieSyncRequest struct {
+	UUID    string   `json:"uuid"`
+	Bidders []string `json"bidders"`
+}
+
+type cookieSyncResponse struct {
+	UUID         string           `json:"uuid"`
+	Status       string           `json:"status"`
+	BidderStatus []*pbs.PBSBidder `json:"bidder_status"`
+}
+
+func cookieSync(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```

Comment:
Any particular reason we use `w` and `r` for var names? Is this convention? (I know it's how the code has been written but I find it more difficult to read)